### ISSUE TITLE
Fix home page to pass accessibility tests

### ISSUE THREAD

**Issue by punamdahiya** [mozilla-services/screenshots#5179]

**Comment by punamdahiya** [comment]
npm run test:accessibility errors out with missing alt tag on img tags  icon-shots and icon-settings. Fix img tags with alt text  to pass accessibility test 

https://github.com/mozilla-services/screenshots/pull/5173#issuecomment-438757834
